1-exercises/C-object-methods/exercise-3.js

```diff
@@ -12,8 +12,8 @@ let highScores = {
   100: ["David"],
 };
 
-let highScoreKeys = /* todo */
-let highScoreValues = /* todo */
+let highScoreKeys = Object.keys(highScores) /* todo */
+let highScoreValues = Object.values(highScores); /* todo */
```

Comment:
Nice job in calling the values, 😉
Althou the `highScoreValues` variables doesn't return the resault.
Please check your code :)